sdk/core/abort-controller/eslint.config.mjs

```diff
@@ -0,0 +1,12 @@
+import azsdkEslint from "@azure/eslint-plugin-azure-sdk";
+
+export default [
+  ...azsdkEslint.configs.recommended,
+  {
+    files: ["test/snippets.spec.ts"],
```

Comment:
Do you think it would be possible to put this config in the ESLint plugin to avoid the duplication?

Reply:
Yes, as per this PR https://github.com/Azure/azure-sdk-for-js/pull/31205